### PR TITLE
feat: replace go-kit/kit with go-kit/log

### PR DESCRIPTION
**Pull request by greut**
It cherry picks what of use.

### CODE REVIEW

**Review by sysadmind — APPROVED** [review]
LGTM I may wait just a bit to merge this so that I can cut a release before making a change like this.

File: go.mod [hunk 1]
```diff
@@ -4,9 +4,19 @@ go 1.16
 
 require (
 	github.com/blang/semver v3.5.2-0.20180723201105-3c1074078d32+incompatible
-	github.com/go-kit/kit v0.10.0
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
```
Comment (SuperQ) [inline go.mod:7]: Please drop these excess `// indirect` requirements.

**Comment by SuperQ** [comment]
@sysadmind This is a pretty simple swap. I would suggest getting it in before the next release.

**Comment by roidelapluie** [comment]
> @sysadmind This is a pretty simple swap. I would suggest getting it in before the next release

:+1:

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**